judge/views/organization.py

```diff
@@ -80,6 +81,16 @@ def get_context_data(self, **kwargs):
         context = super(OrganizationHome, self).get_context_data(**kwargs)
         context['title'] = self.object.name
         context['can_edit'] = self.can_edit_organization()
+        context['is_member'] = self.request.profile in self.object if self.request.user.is_authenticated else False
```

Comment:
IIRC `request.profile` should be `None` if the user is not authenticated, so I don't think the `if self.request.user.is_authenticated` is needed.

Reply:
Checking if the user is authenticated here is necessary, as passing `None` into `Organization.__contains__` causes:
```
  File "/code/site/judge/models/profile.py", line 55, in __contains__
    raise TypeError('Organization membership test must be Profile or primany key')
TypeError: Organization membership test must be Profile or primany key
```
This is from https://github.com/DMOJ/site/blob/master/judge/models/profile.py#L55 

One solution to avoid having to check if the user is authenticated here would be to check if `None` is passed in `Organization.__contains__`, and returning `False`.